### ISSUE TITLE
The cropping part paper mentioned  missed?

### ISSUE THREAD

**Issue by qinwang-ai** [assassint2017/abdominal-multi-organ-segmentation#3]
` # 首先将输入缩小一倍
        inputs_stage1 = F.upsample(inputs, (48, 128, 128), mode='trilinear')

        # 得到第一阶段的结果
        output_stage1 = self.stage1(inputs_stage1)
        output_stage1 = F.upsample(output_stage1, (48, 256, 256), mode='trilinear')

        # 将第一阶段的结果与原始输入数据进行拼接作为第二阶段的输入
        inputs_stage2 = torch.cat((output_stage1, inputs), dim=1)`

The paper "A multi-scale pyramid of 3D fully convolutional networks for abdominal multi-organ segmentation" mentioned "The lower-resolution-level 3D FCN predictions are upsampled, cropped and concatenated with the inputs of a higher resolution 3D FCN."

I think it means the input of first stage not just 0.5 downsampling of the input simply, the input of the first stage should have more visual range than the input of second stage, the Fig.1 in the paper shows this very clearly.
I think the right process is  let the input as first stage's input, then crop the center square area of the input (w, h = 0.5*input_w, 0.5*input_h), then upsample it 2x to the same size of the input, use it as the input of second stage,  then go on......
